integration/js/app/video_test/server.js

```diff
@@ -0,0 +1,120 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+const compression = require('compression');
+const fs = require('fs');
+const http = require('http');
+const url = require('url');
+const { v4: uuidv4 } = require('uuid');
+
+// Store created meetings in a map so attendees can join by meeting title
+const meetingTable = {};
+
+// Use local host for application server
+const host = '127.0.0.1:8080';
+
+// Load the contents of the web application to be used as the index page
+const indexPage = fs.readFileSync(`dist/${process.env.npm_config_app || 'index'}.html`);
+
+// Create ans AWS SDK Chime object. Region 'us-east-1' is currently required.
+// Use the MediaRegion property below in CreateMeeting to select the region
+// the meeting is hosted in.
+const chime = new AWS.Chime({ region: 'us-east-1' });
+
+// Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com.
+chime.endpoint = new AWS.Endpoint(process.env.ENDPOINT || 'https://service.chime.aws.amazon.com');
+
+// Start an HTTP server to serve the index page and handle meeting actions
+http.createServer({}, async (request, response) => {
+  log(`${request.method} ${request.url} BEGIN`);
+  try {
+    // Enable HTTP compression
+    compression({})(request, response, () => {});
+    const requestUrl = url.parse(request.url, true);
+    if (request.method === 'GET' && requestUrl.pathname === '/') {
+      // Return the contents of the index page
+      respond(response, 200, 'text/html', indexPage);
+    } else if (process.env.DEBUG && request.method === 'POST' && requestUrl.pathname === '/join') {
```

Comment:
Can you double check that this work ?

Reply:
Checked it, and it doesnt work for me. We probably dont need this, so Im removing it.